biogeochem/EDMortalityFunctionsMod.F90

```diff
@@ -42,28 +42,35 @@ module EDMortalityFunctionsMod
 
 
 
-  subroutine mortality_rates( cohort_in,bc_in,cmort,hmort,bmort,frmort )
+  subroutine mortality_rates( cohort_in,bc_in,cmort,hmort,bmort,frmort,smort,asmort )
 
     ! ============================================================================
     !  Calculate mortality rates from carbon storage, hydraulic cavitation, 
-    !  background and freezing
+    !  background and freezing and size dependent senescence
```

Comment:
and size _and age_ dependent senescence maybe? 

Also, is it me, or is 'senescence' really, really, hard to spell?

Reply:
Added 'age' in there. And yes! It is a terrible word to have to type repeatedly.

Reply:
Ha ha. Glad it isn't just my brain senescing, or whatever.